scripts/configs/utils/getInputPath.js

```diff
@@ -1,4 +1,4 @@
 import { join } from "node:path";
 import { getFixturesDir } from "./getFixturesDir.js";
 
-export const getInputPath = (version) => join(getFixturesDir(), version, "index.mjs");
+export const getInputPath = (version) => join(getFixturesDir(), version, "index.js");
```

Comment:
What is Parcel's reasoning for this limitation, if there is any? 

Surprising that any bundler would require this.